Firmware/Drivers/STM32/stm32_spi_arbiter.cpp

```diff
@@ -106,19 +124,44 @@ void Stm32SpiArbiter::on_complete() {
     if (!task_list_) {
         return; // this should not happen
     }
+    if(!task_list_->split_tx_rx) {
```

Comment:
In this section, from line 127 to 166, can you change this to be 

```if (task_list_->split_tx_rx && !task_list_->done_tx) {TLE special case} else {"normal" case}```?

Reply:
Sorry I've been caught up with other stuff... yep I can look into it. It sounds like it would still work with the three cases, just by merging the TLE special rx case into the normal case.

Reply:
I didn't care much about the safety word for now, but it wouldn't be difficult to include support for it and connect it with the error codes for the encoder >> axis

Reply:
I've done some work to get the safety word working. You can find it on the branch `pj_temp`. There's something strange happening with the DMA, but it otherwise seems to work.